app/src/main/java/de/westnordost/streetcomplete/quests/summit/SummitMarkingsUtil.kt

```diff
@@ -0,0 +1,48 @@
+package de.westnordost.streetcomplete.quests.summit
+
+import de.westnordost.streetcomplete.data.elementfilter.toElementFilterExpression
+import de.westnordost.streetcomplete.data.osm.geometry.ElementPolylinesGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+
+val COUNTRIES_WHERE_SUMMIT_MARKINGS_ARE_COMMON = NoCountriesExcept(
```

Comment:
Not sure is it a good idea to have both in the same category, there can be noncristian country with summit registers for example, or reverse (Poland may qualify for summit crosses for example - I am not entirely sure)

Reply:
I agree. I guess summit crosses may be present in any Christian country that has mountains and kind of shares the culture to put crosses on them (e.g. maybe Phillippines too? https://en.wikipedia.org/wiki/Mount_Samat#/media/File:Mount_Samat_remote_view.jpg).

Summit registers seem to be more a question of whether mountain hiking as a hobby is widespread.

But in any case, please do not feel pushed to do a worldwide research where one can find summit crosses. Instead, just add those countries where they are well known / where some are already tagged in OSM.

Reply:
Ok. What I did in [`e69d228`](https://github.com/streetcomplete/StreetComplete/pull/4111/commits/e69d228ed3bbc0176f27874d56e8d1ebb46eb9a1):

* separated the quest filters
* enabled the quest in Poland
* disabled the quest in Slovenia, Greece and the US due to missing evidence that summit crosses are a thing there
* updated the sources for the other countries